app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -278,6 +278,11 @@ def sortable_version(version)
         Util::Package.sortable_version(version)
       end
 
+      apipie :method, 'Returns true if Katello Agent infrastructure is enabled on the server'
+      def katello_agent_enabled
```

Comment:
We also have macros which follow the `?` convention. Would this be clearer?
```suggestion
      def katello_agent_enabled?
```